### PR TITLE
Add Rubocop rules for multiline statements

### PR DESCRIPTION
**Pull request by javierm**
## Objectives

* Make multiline statements easier to read
* Make the code more consistent
* Make it easier to follow the "LineLength" rule in the future :wink:

### CODE REVIEW

File: app/components/shared/advanced_search_component.html.erb [hunk 1]
```diff
@@ -25,16 +25,16 @@
             <%= t("shared.advanced_search.from") %> (<%= t("shared.advanced_search.date_placeholder") %>)
           </label>
           <%= text_field_tag "advanced_search[date_min]",
-                              advanced_search[:date_min],
-                              class: "js-calendar" %>
+                             advanced_search[:date_min],
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/components/shared/advanced_search_component.html.erb:28]: This is a false positive; probably due to a bug in pronto-erb_lint. We've disabled ERB's default linters, so the `RequireInputAutocomplete` linter isn't enabled, and we shouldn't be getting this message.

Running `erblint` doesn't report this issue (as expected).

File: app/components/shared/advanced_search_component.html.erb [hunk 2]
```diff
@@ -25,16 +25,16 @@
...
+                             class: "js-calendar" %>
         </div>
         <div class="filter">
           <label for="advanced_search_date_max">
             <%= t("shared.advanced_search.to") %> (<%= t("shared.advanced_search.date_placeholder") %>)
           </label>
           <%= text_field_tag "advanced_search[date_max]",
-                              advanced_search[:date_max],
-                              class: "js-calendar" %>
+                             advanced_search[:date_max],
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/components/shared/advanced_search_component.html.erb:36]: This is a false positive; probably due to a bug in pronto-erb_lint. We've disabled ERB's default linters, so the `RequireInputAutocomplete` linter isn't enabled, and we shouldn't be getting this message.

Running `erblint` doesn't report this issue (as expected).

File: app/views/admin/poll/shifts/_search_officers.html.erb [hunk 3]
```diff
@@ -3,9 +3,9 @@
                method: :get, remote: true do |f| %>
     <div class="input-group">
       <%= text_field_tag :search,
-                          @search,
-                          placeholder: t("admin.poll_shifts.new.search_officer_placeholder"),
-                          id: "search-officers" %>
+                         @search,
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/views/admin/poll/shifts/_search_officers.html.erb:6]: This is a false positive; probably due to a bug in pronto-erb_lint. We've disabled ERB's default linters, so the `RequireInputAutocomplete` linter isn't enabled, and we shouldn't be getting this message.

Running `erblint` doesn't report this issue (as expected).

File: app/views/admin/poll/shifts/_search_officers.html.erb [hunk 4]
```diff
@@ -3,9 +3,9 @@
...
+                         placeholder: t("admin.poll_shifts.new.search_officer_placeholder"),
+                         id: "search-officers" %>
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/views/admin/poll/shifts/_search_officers.html.erb:8]: This is a false positive; probably due to a bug in pronto-erb_lint. We've disabled ERB's default linters, so the `RequireInputAutocomplete` linter isn't enabled, and we shouldn't be getting this message.

Running `erblint` doesn't report this issue (as expected).

File: app/views/relationable/_form.html.erb [hunk 5]
```diff
@@ -12,8 +12,8 @@
   <div class="input-group">
     <div class="input-group-field">
       <%= text_field_tag :url, "",
-                        "aria-describedby": "related_content_help_text",
-                        placeholder: t("related_content.placeholder", url: root_url) %>
+                         "aria-describedby": "related_content_help_text",
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/views/relationable/_form.html.erb:15]: This is a false positive; probably due to a bug in pronto-erb_lint. We've disabled ERB's default linters, so the `RequireInputAutocomplete` linter isn't enabled, and we shouldn't be getting this message.

Running `erblint` doesn't report this issue (as expected).

File: app/views/admin/poll/shifts/_search_officers.html.erb [hunk 6]
```diff
@@ -3,9 +3,9 @@
                method: :get, remote: true do |f| %>
     <div class="input-group">
       <%= text_field_tag :search,
-                          @search,
-                          placeholder: t("admin.poll_shifts.new.search_officer_placeholder"),
-                          id: "search-officers" %>
+                         @search,
+                         placeholder: t("admin.poll_shifts.new.search_officer_placeholder"),
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/views/admin/poll/shifts/_search_officers.html.erb:7]: This is a false positive; probably due to a bug in pronto-erb_lint. We've disabled ERB's default linters, so the `RequireInputAutocomplete` linter isn't enabled, and we shouldn't be getting this message.

Running `erblint` doesn't report this issue (as expected).

File: app/components/shared/advanced_search_component.html.erb [hunk 7]
```diff
@@ -25,16 +25,16 @@
...
+                             class: "js-calendar" %>
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/components/shared/advanced_search_component.html.erb:37]: This is a false positive; probably due to a bug in pronto-erb_lint. We've disabled ERB's default linters, so the `RequireInputAutocomplete` linter isn't enabled, and we shouldn't be getting this message.

Running `erblint` doesn't report this issue (as expected).

File: app/components/shared/advanced_search_component.html.erb [hunk 8]
```diff
@@ -25,16 +25,16 @@
             <%= t("shared.advanced_search.from") %> (<%= t("shared.advanced_search.date_placeholder") %>)
           </label>
           <%= text_field_tag "advanced_search[date_min]",
-                              advanced_search[:date_min],
-                              class: "js-calendar" %>
+                             advanced_search[:date_min],
+                             class: "js-calendar" %>
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/components/shared/advanced_search_component.html.erb:29]: This is a false positive; probably due to a bug in pronto-erb_lint. We've disabled ERB's default linters, so the `RequireInputAutocomplete` linter isn't enabled, and we shouldn't be getting this message.

Running `erblint` doesn't report this issue (as expected).

**Review by taitus — APPROVED** [review]

**Pull request merged**